src/Security/Member.php

```diff
@@ -775,6 +775,7 @@ public function onBeforeWrite()
             && $this->isChanged('Password')
             && $this->record['Password']
             && $this->Email
+            && filter_var($this->Email, FILTER_VALIDATE_EMAIL)
```

Comment:
It can just be this, `filter_var` will correctly handle empty strings and nulls etc.
```suggestion
            && filter_var($this->Email, FILTER_VALIDATE_EMAIL)
```

Reply:
Should probably use `Email::is_valid_address($this->Email ?? '')` instead